makefile

```diff
@@ -8,32 +8,33 @@ CHEATSHEETS   = pymapdl_cheat_sheet pyfluent_cheat_sheet pyaedt_API_cheat_sheet
 all: $(CHEATSHEETS)
 
 pymapdl_cheat_sheet:
-	latexmk -f -pdf -use-make cheat_sheets/mapdl_cheat_sheet/pymapdl_cheat_sheet.tex -cd -outdir=../../$(BUILD_DIR) -interaction=nonstopmode || true
+	latexmk -f -pdf -use-make -outdir=$(BUILD_DIR) cheat_sheets/mapdl_cheat_sheet/pymapdl_cheat_sheet.tex  -interaction=nonstopmode || true
```

Comment:
The generation of the Python scripts should be included as part of the make targets... otherwise, if I haven't executed it before (manually), the build process of the LaTeX file will fail. Include it here.

Reply:
As of now, only "pyfluent" contains the required script, so i added it in pyfluent.